### ISSUE TITLE
[Quick add] Apply equal height to buttons when there’s a taller one

### ISSUE THREAD

**Issue by melissaperreault** [Shopify/dawn#1593]
> If one of the 2 buttons label in [other languages are longer](https://screenshot.click/22-03-hxzo6-vqmcn.png), can we enlarge vertically all of them? It could help enclose the grid structure we are offering with the bottom alignment. In my example, I made "Choose options" longer to stress test this example.

I'm not sure if that is possible. or at least, I don't know of any way to do it with the current HTML markup we have. If other devs have any idea, please let me know.

_Originally posted by @LucasLacerdaUX in https://github.com/Shopify/dawn/issues/1567#issuecomment-1087665372_

**Comment by bredowmax** [comment]
@melissaperreault yes I can recreate this issue: it can happen on mobile, if on the 2-column-layout, when one product has several variants but the other one doesn't have any variants defined.
<img width="602" alt="Screen Shot 2022-04-10 at 6 38 37 AM" src="https://user-images.githubusercontent.com/30573532/162614226-0294bb10-4158-4b92-b822-8cfa930bae7c.png">

Personally, I am not a big friend of the 2-column layout on mobile, and this issue is one reason why. I propose an alternate 1-column layout where image is on the left and product information is on the right. That would mitigate this issue. More here: #1608
